addons/sale_timesheet/static/src/components/so_line_field/so_line_field.xml

```diff
@@ -3,6 +3,7 @@
 
     <t t-name="sale_timesheet.SoLineField">
         <Many2One t-props="m2oProps"/>
+        <span class="text-muted" t-out="props.placeholder" t-if="props.readonly and !props.record.data.so_line"/>
```

Comment:
Cannot we use `falsy_value_label` instead?

This attribute is available in fields definition.